grpcbp/tracing.go

```diff
@@ -44,3 +44,13 @@ func methodSlug(method string) string {
 	split := strings.Split(method, "/")
 	return split[len(split)-1]
 }
+
+// serviceAndMethodSlug splits the UnaryServerInfo.FullMethod and returns
+// the package.service part separate from the method part.
+// ref: https://pkg.go.dev/google.golang.org/grpc#UnaryServerInfo
```

Comment:
The example given by that is `/package.service/method`, which would produce `/package.service/` and `method` as the return values, and using `/package.service/` as the service name sounds weird to me.

Also a better place for this function might be `grpcbp/prometheus.go`?

Also since both return values are strings, please use named returns as a documentation

Reply:
+1 -- I would expect "package.service" and "method" when given "/package.service/method" as input